<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VeroDialogi</class>
 <widget class="QDialog" name="VeroDialogi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arvonlisävero</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Vero&amp;laji</string>
     </property>
     <property name="buddy">
      <cstring>verolajiCombo</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="verolajiCombo"/>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="tyyppiGroup">
      <property name="title">
       <string>Kirjaus on</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="veronalainenRadio">
         <property name="text">
          <string>Veron&amp;alainen määrä</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="veroRadio">
         <property name="text">
          <string>&amp;Veron määrä</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="vahennysRadio">
         <property name="text">
          <string>V&amp;ähennettävä määrä</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="maksuMyyntisaatava">
         <property name="text">
          <string>Maksuperusteinen alv: Myyntisaatavat</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="maksuOstovelka">
         <property name="text">
          <string>Maksuperustainen alv: Ostovelat</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Vero&amp;kanta</string>
      </property>
      <property name="buddy">
       <cstring>prossaSpin</cstring>
      </property>
     </widget>
     <widget class="QSpinBox" name="prossaSpin">
      <property name="suffix">
       <string> %</string>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VeroDialogi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VeroDialogi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maksuMyyntisaatava</sender>
   <signal>toggled(bool)</signal>
   <receiver>verolajiCombo</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maksuOstovelka</sender>
   <signal>toggled(bool)</signal>
   <receiver>verolajiCombo</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maksuMyyntisaatava</sender>
   <signal>toggled(bool)</signal>
   <receiver>prossaSpin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maksuOstovelka</sender>
   <signal>toggled(bool)</signal>
   <receiver>prossaSpin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
